README.md

```diff
@@ -1,5 +1,7 @@
 # GitHub Action for Azure CLI
 
+> [!WARNING]
+> The base image of Azure CLI's docker image is switched to Azure Linux since version `2.64.0`. The `az` commands will remain unaffected. To ensure a smooth transition, please migrate Alpine-specific commands to Azure Linux commands in azure/cli action. For more information, see https://go.microsoft.com/fwlink/?linkid=2282203.

```

Comment:
```suggestion
> Starting with Azure CLI version `2.64.0`, the Azure CLI docker image will be based of Azure Linux. The `az` commands are not affected by this change. To ensure the compatibility of your pipelines, please migrate Alpine-specific commands to Azure Linux commands in the scripts used in the azure/cli action. 
For more information, see https://go.microsoft.com/fwlink/?linkid=2282203.
```

Reply:
I don't think "based of" is correct English: https://www.merriam-webster.com/grammar/based-on-vs-based-off. It should be "based off", but "based on" seems more common.